pkg/systemd/services/service-details.jsx

```diff
@@ -224,6 +253,29 @@ export class ServiceDetails extends React.Component {
         return true;
     }
 
+    doPropertyPolling() {
+        systemd_client.call(this.props.unit.path,
+                            "org.freedesktop.DBus.Properties", "GetAll",
+                            [cockpit.format("org.freedesktop.systemd1.$0", this.unitTypeCapitalized)])
+                .then(result => {
+                    this.addUnitProperties({
+                        MemoryCurrent: (result[0].MemoryCurrent && result[0].MemoryCurrent.v > 0) ? result[0].MemoryCurrent.v : null,
+                        Listen: result[0].Listen ? result[0].Listen.v : null
+                    });
+                });
```

Comment:
Let's please handle errors from GetAll here (with a simple console.log) to avoid a page Oops in weird cases.